src/helpers/xml-builder.js

```diff
@@ -590,6 +590,17 @@ const buildIndentation = (left = 720) => {
   return indentationFragment;
 };
 
+const buildPStyle = (style = 'Normal') => {
```

Comment:
I hope this is based on some actual value and not typed out for namesake :P

Reply:
lol :D No, from what I understand that is the default (See OOXML part 4, page 700). But in some places I saw that "Text" was used so I'm not 100% sure.